web/templates/web/_navigation.html

```diff
@@ -30,8 +30,8 @@
                         {% nav_link request 'web:studies-history' 'My Past Studies' %}
                         {% nav_link request 'logout' logout %}
                     {% else %}
-                        {% nav_signup request 'Sign up' True %}
-                        {% nav_login request 'Login' True %}
+                        {% nav_signup request 'Sign up' %}
+                        {% nav_login request 'Login' %}
```

Comment:
I previously edited the `nav_next` template tag but mixed up the button/link classes, which I fixed in this PR to produce the right styling for the login / signup buttons on the participate page. However this meant removing the `True` argument in the `_navigation` template here so that these nav items are styled correctly. There aren't any other instances of `nav_signup` or `nav_login` that need to be fixed.